### PR TITLE
split parser and notifier

### PR DESCRIPTION
**Pull request by lockwooddev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+* Slack
 
 ## Example usage
 
 ```yaml
-- name: Bandit Notifier
+- name: Parse Bandit output
+  id: bandit-message
+  uses: cbrix-company/action-messages@v1
+  with:
```
Comment (k-shlomi) [inline README.md:14]: missing `outputs`. we should define here the output-file no?

File: notifiers/slack.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    url = 'https://slack.com/api/chat.postMessage'
+    logger.info('POST {0}'.format(url))
+
+    headers = {
+        'Authorization': f'Bearer {token}',
+        'Content-type': 'application/json; charset=utf-8',
+    }
+
+    response = requests.post(url, data=json.dumps(post_body), headers=headers)
+    if response.status_code != 200:
```
Comment (k-shlomi) [inline notifiers/slack.py]: use `HTTPStatus` here

**Review by k-shlomi — COMMENTED** [review]
CR

File: README.md (hunk 1 above)
Comment (lockwooddev) [reply README.md:14]: Still working on it ;)

Comment (lockwooddev) [reply README.md:14]: I missed it and now committing

**Review by k-shlomi — APPROVED** [review]

**Pull request merged**